lib/ex_crypto.ex

```diff
@@ -385,7 +385,10 @@ defmodule ExCrypto do
   end
 
   defp _decrypt(key, initialization_vector, cipher_data, algorithm) do
-    {:ok, :crypto.block_decrypt(algorithm, key, initialization_vector, cipher_data)}
+    case :crypto.block_decrypt(algorithm, key, initialization_vector, cipher_data) do
+      :error -> {:error, "Decryption failed"}
```

Comment:
Can we make this reason an atom instead of a string?
```
:error -> {:error, :decrypt_failed}
```

Reply:
Sure. The type signature changes, so I updated the @spec as well.